### DISCUSSION TITLE
Transmission Rollback

### DISCUSSION THREAD

**Discussion by barbaracrossler** [community/community#46972] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

How do I rollback Transmission to previous version after upgrade?  I have MAC OS Ventura 13.1

**Comment by chris-c-thomas** [comment, marked as answer]
A bit off topic but you can find Transmission's previous releases here: https://github.com/transmission/transmission/releases

Delete the current one and pick the version you want.
